### ISSUE TITLE
Parallel setup_for_alphafold.py ?

### ISSUE THREAD

**Issue by StysPaul** [phbradley/TCRdock#7]
Hello,
The script that prepares the data for prediction, setup_for_alphafold.py, is long when you have a lot of data. I've noticed that it's not parallelized, is it possible to speed it up ?

**Comment by phbradley** [comment]
Hi Paul, I bet that would be possible. For me, the bottleneck is running the predictions, so I haven't put much thought into doing that yet. Believe it or not, it used to be even slower!

**Comment by phbradley** [comment]
Also, note that there is an updated docking protocol and associated fine-tuned parameter set available. You can see an example of how that works in the google-colab jupyter notebook:

https://github.com/phbradley/TCRdock/blob/main/tcrdock_colab_pipeline_v1.ipynb

**Comment by phbradley** [comment]
tldr:
1. update to the newest version of the TCRdock repository
2. add the flag `--new_docking` when running `setup_for_alphafold.py`
3. add the flags `--model_names model_2_ptm_ft4 --model_params_files /path/to/tcrpmhc_run4_af_mhc_params_891.pkl` when running `run_prediction.py`, where that `.pkl` file is the fine-tuned parameter set downloaded from here:

https://www.dropbox.com/s/jph8v1mfni1q4y8/tcrpmhc_run4_af_mhc_params_891.pkl

**Comment by StysPaul** [comment]
Indeed, predictions take much longer than setup. It was a comment to optimize a little.
Thanks for the information about the changes and the new model, I'm currently testing this. By the way, I was hoping that with a new model the calculation time would decrease a bit, but the duration is the same. It's just the precision that's improved?

**Comment by phbradley** [comment]
Good question! The new model just runs a single alphafold simulation per target, rather than running 3 and taking the best score by pMHC:TCR PAE, so in that sense it is 3x faster.
